vcf/vcf.go

```diff
@@ -26,10 +26,10 @@ type Vcf struct {
 }
 
 // GenomeSample is a substruct of Vcf, and contains information about each sample represented in a VCF line.
-// AlleleOne and AlleleTwo are set to -1 if no genotype data is present. AlleleTwo is set to -1 for hemizygous sites.
+// Indexes in Alleles are set to -1 if no genotype data is present.
 type GenomeSample struct {
-	AlleleOne  int16    // First allele in genotype, 0 for reference, 1 for Alt[0], 2 for Alt[1], etc.
-	AlleleTwo  int16    // Second allele in genotype, same Number format as above.
-	Phased     bool     // True for phased genotype, false for unphased.
-	FormatData []string // FormatData contains additional sample fields after the genotype, which are parsed into a slice delimited by colons. Currently contains a dummy empty string in FormatData[0] corresponding to "GT" in Format, so indices in FormatData will match the indices in Format.
+	Alleles    []int16  // Alleles present in genotype, 0 for reference, 1 for Alt[0], 2 for Alt[1], etc.
+	Phase      []int16  // True for phased genotype, false for unphased. len(Phase) == len(Alleles). Phase[0] == true if and only if Phase[1:] == true
```

Comment:
Should this be a []bool instead of a []int16?

Reply:
Yes, fixed.